openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -33,6 +34,14 @@
         )}
     % endif
 
+    % if INCLUDE_PORTFOLIO_UPSELL_MODAL.is_enabled():
```

Comment:
Thanks for the update.  Did you manually ensure the modal appears as you wanted in all the right places?  Just asking because of the earlier missing `is_enabled()` issue.